### DISCUSSION TITLE
Pre-rendered Thunder indicator is gone?

### DISCUSSION THREAD

**Discussion by StewartGF** [vercel/next.js#19410] [Help] [answered]
I just created a Next project with 10.0.1 and pages that have text only are not showing the "thunder Pre-rendered Page" indicator, tried 
```js
module.exports = {
  devIndicators: {
    autoPrerender: true,
  },
};
```
but it doesn't work. 
What can I do?, the feature it's gone for ever ?:(

**Comment by jamesmosier** [comment, marked as answer]
Yes, it was removed in this PR: https://github.com/vercel/next.js/pull/18629

These issues were the reasoning:

* https://github.com/vercel/next.js/issues/18275
* https://github.com/vercel/next.js/issues/18124
* https://github.com/vercel/next.js/pull/18261

**Reply by StewartGF** [reply]
thank you James 😁
